### ISSUE TITLE
SyntaxError parser.version = f"v{__version__}"

### ISSUE THREAD

**Issue by iFloris** [ivandokov/phockup#178]
Hello, 

After manually installing Phockup, I am getting a syntax error. 
Though I have used Phockup on several different machines, this is the first time I have seen this.

```
/usr/local/bin/phockup -v
  File "/usr/local/bin/phockup", line 33
    parser.version = f"v{__version__}"
                                     ^
SyntaxError: invalid syntax
```

The installed python is
Python 3.5.3

```
Description:	TurnKey GNU/Linux 9.13 (stretch)
Release:	9.13
Codename:	stretch
```

What am I doing wrong?

**Comment by unapproachable** [comment]
The syntax that is failing is the "`f`" used in the `parser.version` statement.  This was introduced to Python in version 3.6.  Python 3.5.3 was released in 2017 and Python 3.5 was End Of Life in 2020.  My suggestion would be to get to a more modern release of Python to support the more recent syntaxes and functionality provided in the core package.  Even if you were to get beyond this particular syntax issue, you'd immediately encounter other compatibility issues and dependencies.

I suggest at least Python 3.8, since that has support until 2024.  Python 3.10 may be the best choice for late 2022.

https://www.python.org/downloads/

**Comment by iFloris** [comment]
Thank you!
